travel-event-host-app/src/models/event.ts

```diff
@@ -0,0 +1,28 @@
+import Category from '@/lib/category';
+import mongoose, { Schema, models } from 'mongoose';
+const eventSchema = new Schema(
```

Comment:
We'll need the corresponding typescript interface for User. Then you can do something like new Schema<User>. Maybe you can move schemas to a separate folder called "schemas"?

Reply:
what happen with the folder models?, is deleted?